app/views/movement_confirmation_page.scala.html

```diff
@@ -14,18 +14,35 @@
  * limitations under the License.
  *@
 
-@import forms.Choice
+@import forms.{Choice, ConsignmentReferences}
 @import views.Title
+@import models.requests.JourneyRequest
 
 @this(main_template: views.html.main_template)
 
-@(choice: Choice)(implicit request: Request[_], messages: Messages)
+@(consignmentReferences: ConsignmentReferences)(implicit request: JourneyRequest[_], messages: Messages)
 
-@main_template(title = Title(s"movement.${choice.value}.confirmation")) {
+@main_template(
+    title = Title(s"movement.${request.choice.value}.confirmation.tab.heading")
+) {
 
-  @components.highlight_box_with_reference(
-    headingText = messages(s"movement.${choice.value}.confirmation")
-  )
+    @components.highlight_box_with_reference(
+        headingText = messages(
+            s"movement.${request.choice.value}.confirmation.heading",
+            if(consignmentReferences.reference == "D") "DUCR" else "MUCR",
```

Comment:
Nit: Maybe it would be more clean if you'd extract the logic from there.

Reply:
As discussed, we will stay with this